DROD/TemporalMoveEffect.cpp

```diff
@@ -86,10 +88,17 @@ bool CTemporalMoveEffect::Draw(SDL_Surface* pDestSurface)
 		pDestSurface = GetDestSurface();
 
 	const float percent = (dwElapsed - this->startDelay) / float(this->dwDuration);
-	const Uint8 opacity = static_cast<Uint8>(255 * (1.0f-percent));
+	const float opacityPercent = percent > 0.5
```

Comment:
This var technically appears to be representing 'transparencyPercent', not opacity.